node/src/logger.rs

```diff
@@ -0,0 +1,33 @@
+use env_logger::fmt::Color;
+use std::io::Write as _;
+
+/// Initializes [env_logger] using the `RUST_LOG` environment variables and our custom formatter.
+pub fn init() {
+    let mut builder = env_logger::Builder::from_default_env();
+
+    builder.format(|formatter, record| {
+        let time = time::OffsetDateTime::now_local();
```

Comment:
nit: move the formatting part to a separate function that you'd then pass here or use in the closure:

```suggestion
        builder.format(OurCustomFormatter);
```